<?xml version="1.0"?>

<launch>
    
    <arg name="world_file" value="$(find local_planner_benchmark)/worlds/tricky_maze/tricky_maze.world"/>
    <arg name="world_name" value="tricky_maze"/>
    <arg name="map_file" value="$(find mir_navigation)/map/tricky_maze.yaml"/>
    <arg name="dynamic" default="false"/>
    <arg name="move_arm" default="false"/>
    <arg name="initial_x" value="-8.567309"/>
    <arg name="initial_y" value="2.555640"/>
    <arg name="initial_z" value="0.5"/>
    <arg name="initial_yaw" value="0.0"/>
    <arg name="verbose" value="false"/>
    <arg name="trial_arg" default="1"/>
    <arg name="local_planner" default="dwb"/>

    <include file="$(find local_planner_benchmark)/launch/common_benchmarking.launch" pass_all_args="true">
        <param name="world_name_py" value="$(arg world_name)"/>
    </include>
</launch>
